lua/neogit/status.lua

```diff
@@ -81,6 +83,9 @@ local function get_section_item_for_line(linenr)
   if section == nil then
     return nil, nil
   end
+  if item_idx == nil then
+    return section, nil
+  end
```

Comment:
I assume this just makes sure that there is an item (and not a blankline under the cursor?